apps/frontend/app/layout.tsx

```
@@ -28,15 +29,17 @@ interface RootLayoutProps {
 
 export default function RootLayout({ children }: RootLayoutProps) {
   return (
-    <html>
+    <html suppressHydrationWarning>
```

Biome (2.1.2)

[error] Provide a lang attribute when using the html element.

Setting a lang attribute on HTML document elements configures the languageused by screen readers when no user default is specified.

(lint/a11y/useHtmlLang)